src/Agent/NewRelic/Agent/Core/Logging/LoggerBootstrapper.cs

```diff
@@ -107,33 +108,46 @@ private static LoggerConfiguration ConfigureInMemoryLogSink(this LoggerConfigura
         }
 
         /// <summary>
-        /// Add the Event Log sink if running on .NET Framework
+        /// Configure an Event Log sink if running on Windows. Logs messages at Warning level and above. Intended for
+        /// use during bootstrapping and as a fallback if the file logging sink can't be created.
+        ///
+        /// The Agent will create the event log source if it doesn't exist *and* if the app is running with
+        /// administrator privileges. Otherwise, it will silently do nothing.
+        ///
+        /// It is possible to manually create the event log source in an elevated Powershell window:
+        ///    New-EventLog -LogName "Application" -Source "New Relic .NET Agent"
+        /// 
         /// </summary>
         /// <param name="loggerConfiguration"></param>
         private static LoggerConfiguration ConfigureEventLogSink(this LoggerConfiguration loggerConfiguration)
         {
-#if NETFRAMEWORK
-            const string eventLogName = "Application";
-            const string eventLogSourceName = "New Relic .NET Agent";
-            try
+#if NETSTANDARD2_0
```

Comment:
You can reduce the amount of compiler directives if you did something like the following instead.
```
#if NETSTANDARD2_0
    var isWindows = RuntimeInformation.IsOSPlatform(OSPlatform.Windows);
#else
    var isWindows = true;
#endif

    if (isWindows)
```